.drone.star

```diff
@@ -2443,3 +2449,147 @@ def fixSharedDataPermissions():
         ],
         "depends_on": ["wait-for-oc10"],
     }]
+
+def litmus(ctx, storage):
+    pipelines = []
+
+    default = {
+        "phpVersions": ["7.3", DEFAULT_PHP_VERSION],
```

Comment:
```suggestion
        "phpVersions": [DEFAULT_PHP_VERSION],
```

Only the test-runner might care about the PHP version. The system-under-test does not use PHP. So just run one pipeline for one PHP version.

Note: actually the PHP version is just a "convenience" in order to select some image for `setupForLitmus`. `setupForLitmus` only runs some bash script, it never invokes PHP. So we could even choose some other owncloud/ubuntu or similar image to use to run `setup-for-litmus.sh`